tests/influence/conftest.py

```diff
@@ -75,7 +52,152 @@ def linear_model(problem_dimension: Tuple[int, int], condition_number: float):
     return A, b
 
 
-def create_mock_dataset(
+def linear_derivative_analytical(
+    linear_model: Tuple["NDArray", "NDArray"], x: "NDArray", y: "NDArray"
+) -> "NDArray":
+    """
+    :param linear_model: A tuple of np.ndarray' of shape [NxM] and [N] representing A and b respectively.
+    :param x: A np.ndarray of shape [BxM].
+    :param y: A np.nparray of shape [BxN].
+    :returns: A np.ndarray of shape [Bx((N+1)*M)], where each row vector is [d_theta L(x, y), d_b L(x, y)]
```

Comment:
It is unnecessary to repeat the type in the param docstring, since it's in the function's signature.
```suggestion
    :param linear_model: A tuple of matrices of shape [NxM] and [N] representing A and b respectively.
    :param x: A matrix of shape [BxM].
    :param y: A matrix of shape [BxN].
    :returns: A matrix of shape [Bx((N+1)*M)], where each row vector is [d_theta L(x, y), d_b L(x, y)]
```
Also, it's better to use LaTeX (🙈) for the math. Sphinx renders it and it can be less ambiguous (what is `d_theta L(x,y)`? Better to have the definition of $L(x,y)$ in the docstring to make the dependence on $\theta, b$ clear)